spec/controllers/events_controller_spec.rb

```diff
@@ -167,6 +167,38 @@
     end
   end
 
+  describe "GET #badges" do
+    let(:valid_attributes) { FactoryGirl.attributes_for(:event_with_accepted_applications) }
+
+    it "assigns the requested event as @event" do
+      event = Event.create! valid_attributes
+      get :badges, event_id: event.to_param, session: valid_session
+      expect(assigns(:event)).to eq(event)
+    end
+  end
+
+  describe "POST #badges" do
+    it "contains two name badges with title 'Max Mustermann'" do
+      event = Event.create! valid_attributes
+      rendered_pdf = post :print_badges,
+                          event_id: event.to_param,
+                          session: valid_session,
+                          "1234_print_Max Mustermann"  => 1,
+                          "1235_print_Max Mustermann"  => 1,
+                          "1236_print_Max Mustermann"  => 1,
+                          "1237_print_Max Mustermann"  => 1,
+                          "1238_print_Max Mustermann"  => 1,
+                          "1239_print_Max Mustermann"  => 1,
+                          "1240_print_Max Mustermann"  => 1,
+                          "1241_print_Max Mustermann"  => 1,
+                          "1242_print_Max Mustermann"  => 1,
+                          "1243_print_Max Mustermann"  => 1,
+                          "1244_print_Max Mustermann"  => 1
+      pdf = PDF::Inspector::Text.analyze(rendered_pdf.body)
+      expect(pdf.strings).to include("Max Mustermann")
```

Comment:
Maybe add a few different names names to make sure that more than one gets printed :smile: